operator/cmd/mesh/testdata/manifest-generate/input/istio_csr.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  namespace: istio-system
+spec:
+  meshConfig:
+    trustDomain: cluster.local
+  values:
+    global:
+      caAddress: cert-manager-istio-csr.cert-manager.svc:443
+    pilot:
+      env:
+        ENABLE_CA_SERVER: false
+      extraContainerArgs:
+        - --tlsCertFile=/etc/cert-manager/tls/tls.crt
```

Comment:
Any reason to not use the build-in defaults ( by mounting the volume in the expected location ) ? 

The '/var/run/secret' is preferred for consistency and because (some) tools know to skip it.